### PR TITLE
SVG symbol sprite documentation & cleanup

### PR DESCRIPTION
**Pull request by jpc-ae**
Full documentation added for SVG symbol usage. Existing documentation was also cleaned up somewhat in the process. This is a replacement for the previous merge that included my own generated symbol files. Note that '_24px' is not needed in the ids since they are size independent; the viewbox just states where the path numbers sit relative to. This could maybe be removed when generating the files (and the documentation updated accordingly).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,101 +6,121 @@
     <link href="//fonts.googleapis.com/css?family=Roboto:400,500,700,400italic" rel="stylesheet" type="text/css">
     <link href="//fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" type="text/css">
     <link rel="stylesheet" type="text/css" href="www/css/material.css">
-    <title>Material icons guide - Google design</title>
+
```
Comment (shyndman) [inline index.html]: This file is generated from Markdown on our end. I've captured your changes, and I'll port them back. Mind reverting this file?

**Comment by shyndman** [comment]
Dude! Amazing work. Thank you.

Revert index and it's ready to go.

**Comment by jpc-ae** [comment]
Got it, all done. Glad to contribute!

FYI, I think the accessibility icon looks too much like a person sitting on a toilet or something. Might want to revisit that one at some point. The gavel too looks a little too abstract to me, but I can live with it.

**Comment by jpc-ae** [comment]
Wait, do I need to revert the CSS file as well?

**Comment by shyndman** [comment]
Never noticed that about the accessibility icon. I'll bring it up. :)

Yes. Please revert the CSS as well.

**Comment by jpc-ae** [comment]
Okay, I've reverted that file as well. Please note that the three statements I had added were for the sake of the symbol samples I included in index.html, so you may need to include them again in some way.

**Pull request merged**